docker/scripts/merge_env.py

```diff
@@ -74,6 +82,10 @@ def _merge_to_env_file(
             if key not in PREFERRED_BASE_ENV_KEYS and key in target_env:
                 value = target_env.get(key, value)
 
+            # Set default value for these keys always.
+            if not value and key in SET_DEFAULT_KEYS:
+                value = SET_DEFAULT_KEYS[key]
```

Comment:
@hari-kuriakose usually people might not change this but will their values get overwritten whenever they merge env?

Reply:
@chandrasekharan-zipstack No, only when a user has an existing setup and this key was added in a later release. In this case, for example if `SYSTEM_ADMIN_USERNAME` is newly added in base nev (`sample.env`) but still empty at this point, then we need to set a default value as this key is marked by `SET_DEFAULT_KEYS` list.

For new setup, it is handled in the `run-platform.sh` script itself as `.env` is created fresh from `sample.env` for the very first time.